samples/drivers/led_gpio/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _gpio-led-sample:
+
+LED GPIO Sample
+###############
+
+Overview
+********
+
+The sample assumes that the LEDs are connected to GPIO lines.
+It is configured to work on boards that have defined the led0, led1, led2, led3
```

Comment:
change to ``led2, and led3 aliases``

Reply:
Ok. I removed led2 and led3. Sample can handle two leds now to ensure compatibility with more boards.